ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -264,15 +283,19 @@ void PowerSystemNode::initAndRun()
     // corresponding prognoser via its message bus.
     // Save the current average mechanical power, in case jointStatesCb runs
     // while processing which would update m_mean_mechanical_power mid-loop.
-    double mech_power = m_mean_mechanical_power;
+    const double actual_mech_power
+      = m_mean_mechanical_power / m_mechanical_efficiency;
```

Comment:
Outside the scope of this PR, but I'd like to know if `m_mean_mechanical_power` is properly computed from joint states.

Reply:
This was an optimization I suggested a ways back. I believe Liam's implementation is correct. 
https://github.com/nasa/ow_simulator/pull/352#discussion_r1289233696